### ISSUE TITLE
SyntaxError: invalid syntax

### ISSUE THREAD

**Issue by ip7571** [python-amazon-mws/python-amazon-mws#148]
Importing MWS package was running fine and I had a few package updates then I started getting this invalid syntax issue. 

```python
Traceback (most recent call last):

  File "/Users/name/anaconda3/lib/python3.7/site-packages/IPython/core/interactiveshell.py", line 3296, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)

  File "<ipython-input-2-bcbf2e2a5a3d>", line 4, in <module>
    from mws import mws

  File "/Users/name/anaconda3/lib/python3.7/site-packages/mws/mws.py", line 204
    except HTTPError, e:
                    ^
SyntaxError: invalid syntax
```

**Comment by Bobspadger** [comment]
Can you confirm what version of MWS you are using?

Can you set up a new virtual environment and attempt to import mws into that?

**Comment by Bobspadger** [comment]
I've just tried the following:

1. Created a new python 3.7.1 virtual environment with pyenv
1. installed mws from pypi with pip
1. imported

All ok:

```python
pip install mws
Collecting mws
  Downloading https://files.pythonhosted.org/packages/07/5e/8798f20a866e761e6d675cfa8dc2591741cec22ed74adf2597398d0818fa/mws-0.8.7-py3-none-any.whl
Collecting requests (from mws)
  Using cached https://files.pythonhosted.org/packages/51/bd/23c926cd341ea6b7dd0b2a00aba99ae0f828be89d72b2190f27c11d4b7fb/requests-2.22.0-py2.py3-none-any.whl
Collecting chardet<3.1.0,>=3.0.2 (from requests->mws)
  Using cached https://files.pythonhosted.org/packages/bc/a9/01ffebfb562e4274b6487b4bb1ddec7ca55ec7510b22e4c51f14098443b8/chardet-3.0.4-py2.py3-none-any.whl
Collecting urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 (from requests->mws)
  Using cached https://files.pythonhosted.org/packages/b4/40/a9837291310ee1ccc242ceb6ebfd9eb21539649f193a7c8c86ba15b98539/urllib3-1.25.7-py2.py3-none-any.whl
Collecting certifi>=2017.4.17 (from requests->mws)
  Using cached https://files.pythonhosted.org/packages/b9/63/df50cac98ea0d5b006c55a399c3bf1db9da7b5a24de7890bc9cfd5dd9e99/certifi-2019.11.28-py2.py3-none-any.whl
Collecting idna<2.9,>=2.5 (from requests->mws)
  Using cached https://files.pythonhosted.org/packages/14/2c/cd551d81dbe15200be1cf41cd03869a46fe7226e7450af7a6545bfc474c9/idna-2.8-py2.py3-none-any.whl
Installing collected packages: chardet, urllib3, certifi, idna, requests, mws
Successfully installed certifi-2019.11.28 chardet-3.0.4 idna-2.8 mws-0.8.7 requests-2.22.0 urllib3-1.25.7
You are using pip version 10.0.1, however version 19.3.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
(venv371)  ~/Downloads/temp/mws_test  python3
Python 3.7.1 (default, Jan 14 2020, 09:43:53)
[Clang 11.0.0 (clang-1100.0.33.16)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from mws import mws
```

**Comment by Bobspadger** [comment]
I've also just looked through the codebase and I can't find an instance of `except HTTPError, e:`

The only reference to the use of HTTPError in the mws code is in the `make_request` method, (both master and develop branches) and its 

`except HTTPError as exc:`

Have you made an accidental change to the code?

**Comment by GriceTurrble** [comment]
The latest occurrence of that Python 2 style `except` clause was v0.6. Later versions correct that syntax, which is not workable in Python 3.

Please upgrade to the latest available using `pip install -U mws<1.0` (that range ensures that the eventual 1.0 release, which will have some backwards-incompatibility, does not break your current usage).
